src/BridgeIHMCORS.cpp

```diff
@@ -148,17 +172,37 @@ void BridgeIHMCORS::run()
                     m_robotFeedback.jointStates()[jnt].q() = m_jointPositionsFromYARP[jnt];
                     m_robotFeedback.jointStates()[jnt].qd() = m_jointVelocitiesFromYARP[jnt];
                 }
-                m_robotFeedback.jointStates()[jnt].tau() = 0.0;
+               
+               m_robotFeedback.jointStates()[jnt].tau() = m_jointTorquesFromYARP[jnt];
+               // std::cerr << "m_robotFeedback message updated " << std::endl;
+            } 
 
-                // std::cerr << "m_robotFeedback message updated " << std::endl;
-            }
 
+           // serialize m_robotFeedback
+           Cdr m_robotFeedback_ser(fastBuffer);
+           m_robotFeedback_ser << m_robotFeedback;
+
+           // open socket
+           udp::socket socket(io_srv);
+           socket.open(udp::v4());
```

Comment:
I am probably missing something: are you opening the socket at every loop?

Reply:
Actually, I tried to move these two lines in the header but I got an error. I will try again and debug.